docs/ldml/tr35-personNames.md

```diff
@@ -41,26 +41,929 @@ The LDML specification is divided into the following parts:
+
+| Modifier   | Description |
+| ---------- | ----------- |
+| informal   | Requests an informal version of the name if available. For example, {given} might be “Thomas”, and {given-informal} might be “Tom”. If there is no informal version, then the normal one is returned. An informal version should not be generated, because they vary too much: Beth, Betty, Betsy, Bette, Liz, … |
+| prefix     | Return the “prefix” name, or the “tussenvoegsel'' if present. For example, “van der Poel” becomes “van der”, “bint Fadi” becomes “bint”, “di Santis” becomes “di”. Note that what constitutes the prefix is language- and locale-sensitive. It may be passed in as part of the PersonName object, similar to the “_-informal_” modifier, e.g. as “_surname-prefix_”.<br><br>The implementation of this modifier depends on the PersonName object. CLDR does not currently provide support for automatic identification of tussenvoegsels, but may in the future.<br><br>If the resulting “_-prefix_” value is empty, it defaults to an empty string.<br><br>An example sorting pattern for “Johannes van den Berg” may be<br>{surname-core}, {given} {given2} {surname-prefix}<br><br>Only the “-_prefix_” or the “-_core_” modifier may be used, but not both. They are mutually exclusive. |
+| core       | Return the “core” name, removing any tussenvoegsel. For example, “van der Poel” becomes “Poel”, “bint Fadi” becomes “Fadi”, “di Santis” becomes “Santis”. Note that what constitutes the core is language- and locale-sensitive.<br><br>The implementation of this modifier depends on the PersonName object. CLDR does not currently provide support for identification of tussenvoegsel, but may in the future.<br><br>If the resulting “_-core_” value is empty, it defaults to the field it modifies. E.g., if “_surname-core_” is empty in the PersonName object to be formatted, it will default to the “_surname_” field.<br><br>Vice-versa, if the _surname _field is empty, the formatter will attempt to use _surname-prefix_ and _surname-core_, if present, to format the name.<br><br>Only the “-_prefix_” or the “-_core_” modifier may be used, but not both. They are mutually exclusive. |
+| allCaps    | Requests the element in all caps, which is desired In some contexts. For example, a new guideline in Japan is that for the Latin representation of Japanese names, the family name comes first and is presented in all capitals. This would be represented as<br>“{surname-allCaps} {given}”<br><br>Hayao Miyazaki (宮崎 駿) would be represented in Latin characters in Japan (ja-Latn-JP) as “_MIYAZAKI Hayao_”<br><br>_The default implementation uses the default Unicode uppercase algorithm; if the PersonName object being formatted has a locale, and CLDR supports a locale-specific algorithm for that locale, then that algorithm is used. The PersonName object can override this, as detailed below._<br><br>Only the “-_allCaps_” or the “-_initalCap_” modifier may be used, but not both. They are mutually exclusive. |
```

Comment:
check for instances of ", as detailed below." In this case, it isn't needed. In others, may want to point to a part of the document.

Reply:
Also, we probably want to replace "name object" by "PersonName object"

Reply:
For ", as detailed below." it was just 3 places in this table which referred to steps immediately following the table, so I did not add a pointer . I still could in a future version I guess, if it seems warranted.